src/applications/_mock-form/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import OMBInfo from '@department-of-veterans-affairs/component-library/OMBInfo';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+
+import { focusElement } from 'platform/utilities/ui';
+
+class IntroductionPage extends React.Component {
+  componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
+  }
+
+  render() {
+    return (
+      <div className="schemaform-intro">
+        <FormTitle
+          title="Mock Form"
+          subTitle="Equal to VA Form 00-1234 (Mock Form)"
+        />
+        <SaveInProgressIntro
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
+          verifyRequiredPrefill={false}
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          startText="Start the Application"
+        >
+          Please complete the 00-1234 form to apply for Mock form.
+        </SaveInProgressIntro>
+        <h4>Follow the steps below to apply for Mock form.</h4>
+        <div className="process schemaform-process">
+          <ol>
+            <li className="process-step list-one">
+              <h5>Prepare</h5>
+              <h6>To fill out this application, you’ll need your:</h6>
+              <ul>
+                <li>Social Security number (required)</li>
+              </ul>
+              <p>
+                <strong>What if I need help filling out my application?</strong>{' '}
+                An accredited representative, like a Veterans Service Officer
+                (VSO), can help you fill out your claim.{' '}
+                <a href="/disability-benefits/apply/help/index.html">
+                  Get help filing your claim
+                </a>
+              </p>
+            </li>
+            <li className="process-step list-two">
+              <h5>Apply</h5>
+              <p>Complete this Mock form form.</p>
+              <p>
+                After submitting the form, you’ll get a confirmation message.
+                You can print this for your records.
+              </p>
+            </li>
+            <li className="process-step list-three">
+              <h5>VA Review</h5>
+              <p>
+                We process claims within a week. If more than a week has passed
+                since you submitted your application and you haven’t heard back,
+                please don’t apply again. Call us at.
+              </p>
+            </li>
+            <li className="process-step list-four">
+              <h5>Decision</h5>
+              <p>
+                Once we’ve processed your claim, you’ll get a notice in the mail
+                with our decision.
+              </p>
+            </li>
+          </ol>
+        </div>
```

Comment:
We're trying to discourage this, so could this use `<va-process-list>` as an example of a best practice?